### ISSUE TITLE
32 bit build

### ISSUE THREAD

**Issue by hhirsch** [spolu/breach_core#102]
Looking forward to a 32 bit build for my legacy hardware.

**Comment by spolu** [comment]
Someone working on it on IRC... you should probably ping him ZeusTraining there

**Comment by thinklinux** [comment]
+1

**Comment by jurf** [comment]
+1 too.

**Comment by spolu** [comment]
@mbullington yours? :)

**Comment by mbullington** [comment]
I was working on it, yeah. Can you still distribute the builds that use dynamic linking?

My laptop's memory gets exausted (swap too) using the default builds. 3GB of RAM isn't a lot for building Chromium.

**Comment by spolu** [comment]
Not really. That wouldn't work on many distributions

**Comment by giovaneliberato** [comment]
+1 :)

**Comment by mrhaw** [comment]
+1 :)

**Comment by ghost** [comment]
+1 too

When are you going to release it?[Feeling excited on customizing it XD]

**Comment by zackp30** [comment]
@ericx19 Usually in any project people can't really estimate how long the project will take to be released. :P

**Comment by Bloodyaugust** [comment]
+1
Very excited to participate in some hardcore awesomeness.

**Comment by spolu** [comment]
OK guys. You win. Spawning EC2 instance to build 32bit.

**Comment by spolu** [comment]
Hi guys. added a 32bit build of the exo_browser. Can someone test it?
https://github.com/breach/releases/tree/master/exo_browser/v0.6.1916

See #60 for workaround at startup!

I'll package a Breach release if someone confirms it works!

**Comment by hhirsch** [comment]
On Debian 7:

./exo_browser: /lib/i386-linux-gnu/i686/cmov/libc.so.6: version `GLIBC_2.15' not found (required by ./exo_browser)

**Comment by jurf** [comment]
@spolu works great, Ubuntu 14.04 32 bit. To start it I had to use `--disable-setuid-sandbox` though.

**Comment by spolu** [comment]
Yes this is true of any linux distro... will make things easier and add a wiki page to cover that

**Comment by spolu** [comment]
Also will make a proper breach build linkable from the home page!

**Comment by spolu** [comment]
Partially solved by the availability of exo_browser in 32bit.
